From: Jan-Ivar Bruaroey <wingboy@gmail.com>
Date: Fri, 23 Jun 2023 11:27:20 -0400
Subject: (tmp-cherry-pick) Revert "Remove interim .h files from
 rtc_media_base" (2d57f38c1c)

This reverts commit 16205b31bffba4841aafdeaef1386f52fefe6e77.

Reason for revert: Not all references updated

Original change's description:
> Remove interim .h files from rtc_media_base
>
> Depends on https://chromium-review.googlesource.com/c/chromium/src/+/4120255
> landing first.
>
> Bug: webrtc:14775
> Change-Id: If197ed8f01b0febb92f0d214f5b88c7d325190da
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/288581
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Commit-Queue: Harald Alvestrand <hta@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#39392}

Bug: webrtc:14775
Change-Id: I09758d4179f1ae3e0d2460906f0c0659dc1f1d18
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/295220
Commit-Queue: Harald Alvestrand <hta@webrtc.org>
Auto-Submit: Harald Alvestrand <hta@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#39393}
---
 media/BUILD.gn | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/media/BUILD.gn b/media/BUILD.gn
index 5e75a9ee25..2e5c0bb872 100644
--- a/media/BUILD.gn
+++ b/media/BUILD.gn
@@ -128,8 +128,16 @@ rtc_library("rtc_media_base") {
     "base/adapted_video_track_source.cc",
     "base/adapted_video_track_source.h",
     "base/audio_source.h",
+    "base/codec.h",  # temporary - bugs.webrtc.org/14775
+    "base/delayable.h",  # temporary - bugs.webrtc.org/14775
+    "base/media_channel.h",  # temporary - bugs.webrtc.org/14775
+    "base/media_constants.h",  # temporary - bugs.webrtc.org/14775
     "base/media_engine.cc",
     "base/media_engine.h",
+    "base/rid_description.h",  # temporary - bugs.webrtc.org/14775
+    "base/rtp_utils.h",  # temporary - bugs.webrtc.org/14775
+    "base/stream_params.h",  # temporary - bugs.webrtc.org/14775
+    "base/turn_utils.h",  # temporary - bugs.webrtc.org/14775
     "base/video_adapter.cc",
     "base/video_adapter.h",
     "base/video_broadcaster.cc",
-- 
2.37.1 (Apple Git-137.1)

